c31-swynlake/reactive-nutshell
# Post Task, Friends List.

# Description

Added Task edit functionality. Added friends section to list current friends, remove friends, search for users and add them as friends. 

Applies to issue # 2 & 8

## Type of change

Please delete options that are not relevant.




# Testing Instructions

1. git fetch and checkout newNU
1. copy JSON example file for the tasks, friends, and connections sections.
1. check that home page renders user's tasks, and that edit button allows user to edit and submit edited task. Check that form fields are prerendered, and that complete task allows task to be removed.
1. view friends page, test remove friend button. use search to find other users and add to friends list.



# Checklist:
